src/Domain/CoordinateMaps/CoordinateMap.hpp

```diff
@@ -626,6 +626,27 @@ make_vector_coordinate_map_base(Arg0&& arg_0,
   return return_vector;
```

Comment:
Change your commit message, this is not the correct use of the word `vectorizes` in a code. I literally have no idea what it even means. and why are there what appears to be random blank lines?

Reply:
Previously one could only compose with one map at a time (say, wedge with rotation with translation). I then had to do this for each wedge I wanted to rotate and translate. I've found that I want to rotate multiple wedges with rotations and translations in a single function call, by passing a std::vector of wedges as opposed to a single wedge. Should I change the commit message to "The user can now use make_vector_coordinate_map_base to compose a std::vector of maps with a variadic amount of maps"?

Reply:
Yep, that sounds better but I think you want `to compose each map in a std::Vector of maps with a variadic...`